### ISSUE TITLE
Project Review

### ISSUE THREAD

**Issue by albemlee** [npayanoff/date-a-scientist#1]
### Criteria 1: Valid Python Code
* _Score Level:_ 4/4
* _Comment(s):_ Code included runs without any errors.
 
### Criteria 2: Exploration of Data
* _Score Level:_ 4/4
* _Comment(s):_ Data is explored significantly, and the experimental question(s) chosen are logical and based on the data exploration. Features chosen to answer the question make sense.

### Criteria 3: Machine Learning Techniques used correctly
* _Score Level:_ 4/4
* _Comment(s):_ Algorithms are used correctly and the correct conclusions are drawn from the results.

Notes:
- Think about the times it takes to train a model and generate a prediction separately. If a model takes a very long time to train but generates predictions quickly, would you choose it over a model that trains quickly but takes longer to generate predictions?
- Great job reporting precision and recall in addition to accuracy (I recommend checking out [sklearn.metrics.classification_report](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.classification_report.html))
- You should report both training and test scores to ensure your models did not overfit.
- Check out [sklearn pipelines](https://www.kaggle.com/baghern/a-deep-dive-into-sklearn-pipelines). 

### Criteria 4: Report: Are conclusions clear and supported by data?
* _Score Level:_ 4/4
* _Comment(s):_ Question(s) are stated clearly. The results of 2 regression algorithms and 2 classification algorithms are shown. Conclusions are clearly stated and based on evidence.

### Criteria 5: Code formatting
* _Score Level:_ 4/4
* _Comment(s):_ Code is formatted clearly and readable.

Note:
- Your python script was very easy to follow. I encourage you to try using Jupyter notebooks (ipynb) next time because it allows you to display your thought process and data visualizations alongside your code.
 
### Overall Score: 20/20
Good job on the project. I was amazed that you could predict sex based on word choice. View my notes above for constructive feedback.
